backend/core/views.py

```diff
@@ -293,3 +298,31 @@ def competence_reviews_outstanding(self, request, pk=None):
 #             result[team.id]["permissions"].append(permission)
 
 #     return Response(result)
+
+
+@staff_member_required
+def bulk_add_users_to_team(request, team_id):
+    team = get_object_or_404(Team, pk=team_id)
+
+    if request.method == "POST":
+        form = BulkAddUsersForm(request.POST)
+        if form.is_valid():
+            email_addresses = form.cleaned_data["email_addresses"]
+            users_added_to_team = add_users_to_team(team, email_addresses)
+            if users_added_to_team:
+                messages.success(
+                    request,
+                    f"The following users were successfully added to the \"{team.name}\" team: {', '.join(users_added_to_team)}",
+                )
+                return redirect(f"/admin/core/team/{team_id}/change")
```

Comment:
Use `reverse` or `reverse_lazy` to the named route so that this won't break in case of any changes made to the models. Like you did in the templates:

`<a href="{% url 'admin:app_list' app_label='core' %}">Core</a>`